package.json

```diff
@@ -19,5 +19,10 @@
     "android"
   ],
   "author": "Jack Hsu <jack.hsu@gmail.com> (http://jaysoo.ca/)",
-  "license": "ISC"
+  "license": "ISC",
+  "rnpm": {
+    "android": {
+      "packageInstance": "new ExtraDimensionsPackage(this)"
```

Comment:
after this PR https://github.com/jaysoo/react-native-extra-dimensions-android/pull/10

this line should be only "new ExtraDimensionsPackage()"